uint8 id_kobuki
uint8 id_task
uint8[] port_priority
